tests/queries/0_stateless/01324_insert_tsv_raw.sql

```diff
@@ -0,0 +1,7 @@
+drop table if exists tsv_raw;
+create table tsv_raw (a String, b Int64) engine = Memory;
+insert into tsv_raw format TSVRaw "a 	1
```

Comment:
The following works fine with `TSV` too:
```
$ echo -e "\"a\t1" | clickhouse-local --input-format TSV -S "a String, b Int64" -q "select * from table"
"a	1
```
If I understand correctly, the difference between `TSV` and `TSVRaw` is that `TSVRaw` does not replace escape sequences when parsing input, for example:
```
$ echo -e "\"\\\ta\t1" | clickhouse-local --input-format TSV -S "a String, b Int64" -q "select * from table"
"\ta	1
$ echo -e "\"\\\ta\t1" | clickhouse-local --input-format TSVRaw -S "a String, b Int64" -q "select * from table"
"\\ta	1
```
Could you please add more complex tests?